streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedCachingPersistentWindowStoreTest.java

```diff
@@ -1235,4 +1220,15 @@ private int addItemsToCache() {
         return i;
     }
 
+    private void verifyAndTearDownCloseTests() {
+        verify(underlyingStore).close();
+        verify(cache).flush(CACHE_NAMESPACE);
+        verify(cache).close(CACHE_NAMESPACE);
+
+        // resets the mocks created in #setUpCloseTests(). It is necessary to
+        // ensure that @After works correctly.
+        reset(cache);
+        reset(underlyingStore);
```

Comment:
You do not need this if you use add a `doNothing()` where you specify the exceptions to be thrown. For example, 
```java
doThrow(new RuntimeException("Simulating an error on flush2")).when(cache).flush(CACHE_NAMESPACE);
```
becomes

```java
doThrow(new RuntimeException("Simulating an error on flush2")).doNothing().when(cache).flush(CACHE_NAMESPACE);
```